client/components/PostForm.js

```diff
@@ -80,7 +79,7 @@ class PostForm extends React.Component {
       })
       .then(res => {
         //will re-test this post when route is created
-        axios.post(`/api/sub/${this.state.subredditId}/post`, { title: titleText, type: this.state.type, body: bodyText })
+        axios.post(`/api/sub/${this.state.subredditId}`, { title: titleText, type: this.state.type, body: bodyText })
```

Comment:
This should be good, for testing for now though, only submit posts of type "Text", since I haven't accounted for Images and Videos, which need me to add a "Link" to each post I save.

Reply:
Gotcha